src/fdo_magic/builtin/runtime.rs

```diff
@@ -15,6 +15,7 @@ fn search_paths(filename: &str) -> Vec<PathBuf> {
     let mut search_paths = vec![
         PathBuf::from("/usr/share/mime").join(filename),
         PathBuf::from("/usr/local/share/mime").join(filename),
+        PathBuf::from("//opt/homebrew/share/mime").join(filename),
```

Comment:
this has an extra forward slash on the root path sorry chap @mbrubeck 

probably worth catching if you've not cut a release yet :)

Reply:
Already fixed in https://github.com/mbrubeck/tree_magic/commit/b201eb0e5e6d6efb2ac6d09144de95d604e90220

Reply:
ty!